src/github.com/Lepkem/traefik-plugin-response-code-override/rcoverride_test.go

```diff
@@ -0,0 +1,35 @@
+package traefik_plugin_response_code_override
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func TestOverrideWithoutBody(t *testing.T) {
+
+	req, err := http.NewRequest("GET", "/health-check", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	rr := httptest.NewRecorder()
+
+	// create a handler to use as "next" which will verify the request
+	nextHandler := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+	})
+
+	plugin := &responseCodeOverride{
+		overrrides: map[int]int{200: 429},
+		next:       nextHandler,
+		removeBody: true,
+	}
+
+	// create the handler to test, using our custom "next" handler
+	plugin.ServeHTTP(rr, req)
+
+	if status := rr.Code; status == http.StatusOK {
```

Comment:
We need to be able to assert properly. Look for alternative packages we could use for that ?